.github/ISSUE_TEMPLATE/bug-report-general.md

```diff
@@ -30,7 +30,7 @@ Please note how you installed it (`apt`, `dnf`, `pacman`, source, etc):
 Please provide the output of the fwupd devices recognized in your system.
 
 ```shell
-$ fwupdmgr get-devices --show-all-devices
+$ fwupdmgr get-devices --show-all
```

Comment:
ugh, so this is going to be problematic being in the template - most people have 1.4.x or less and won't have `--show-all` to your point.

Reply:
I wasn't going to update the template; did I?

Reply:
I did! not what i wanted!

Reply:
Okay, fixed.